server/routes/article.routes.js

```diff
@@ -35,6 +35,9 @@ router.post('/:articleId/claps', signInAuth, tryCatch(ClapController.createClap)
 router.get('/', tryCatch(ArticleController.getAllArticles));
 
 router.post('/:articleId/bookmark', signInAuth, tryCatch(createOrRemoveBookmark));
```

Comment:
Hi, @davidokonji I am not quite sure this route follows RESTful conventions. A bookmark belongs to a user so I think this route can be better represented by `/user/userID/bookmarks`

Reply:
@cupidcoder Thanks, the userId should not be passed in the parameters of the URL, would change the route to a User route instead of an article route, its good to say `/user/bookmarks/` suites it better

Reply:
@davidokonji That's very valid. 👍